.changesets/fix_router-external-extensibility-deprecation.md

```diff
@@ -0,0 +1,3 @@
+###  Deprecating the external extensibility (PR #6791)
+
+ We have now deprecated the external extensibility and now have updated error log statements to support coprocessor. By [@briannafugate408](https://github.com/briannafugate408)
```

Comment:
Phrasing: The way this is written sounds like we are deprecating external extensibility.  This is **not** true and should be adjusted.

All this PR does is use the correct name for a feature that we are **not** deprecating, which is the feature formerly known as "external extensibility". 

I broadcast a similar message in https://github.com/apollographql/router/pull/6791#issuecomment-2737225414.